src/reducers/communities.js

```diff
@@ -61,7 +62,11 @@ export default function (state = {}, action) {
     case REMOVE_COMMUNITY_MODERATOR_PENDING:
       community = state[meta.slug]
       moderators = community.moderators
-      return {...state, [meta.slug]: {...community, moderators: filter(moderators, m => m.id !== meta.moderatorId)}}
+      return {...state, [meta.slug]: {...community, moderators: moderators && filter(moderators, m => m.id !== meta.moderatorId)}}
+    case REMOVE_COMMUNITY_MEMBER_PENDING:
```

Comment:
can you combine this and the above case? code's the same except for the name of the variable containing the removed member's id